app/pods/controller-injections-1/parent-component.js

```diff
@@ -12,7 +12,7 @@ export default Component.extend({
       Parent data: {{this.data}}
     </p>
 
-    {{yield}}
+    {{yield data}}
```

Comment:
It is recommended to use `this.data` instead of `data`, so that it is obvious where the value comes from.